src/cli.ts

```diff
@@ -9,7 +9,7 @@ import { calculate, calculateMeasureReports, calculateRaw } from './Calculator';
 program
   .option('-o, --output-type <type>', 'type of output, "raw", "detailed", "reports"', 'detailed')
   .requiredOption('-m, --measure-bundle <measure-bundle>', 'path to measure bundle')
-  .requiredOption('-p, --patient-bundle <patient-bundle>', 'path to  patient bundle')
+  .requiredOption('-p, --patient-bundles <patient-bundles...>', 'paths to patient bundle')
```

Comment:
This may be a silly question, but what does this look like in calling the CLI? Is it -p path/to/patient1.json,path/to/patient2.json? -p path/to/patient1.json -p path/to/patient2.json? Something else?

Reply:
`-p path/to/patient1.json path/to/patient2.json`. So it's backwards compatible with how we have been calling it for the one.

Reply:
It makes use of `commander`'s variadic options. https://www.npmjs.com/package/commander#variadic-option

Reply:
Ah, cool. Once we have anything at all in the README, we can document it there.